adapter/service/outlvlswitch/service.go

```diff
@@ -0,0 +1,184 @@
+package outlvlswitch
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/futurehomeno/cliffhanger/adapter"
+	"github.com/futurehomeno/cliffhanger/adapter/cache"
+	"github.com/futurehomeno/fimpgo"
+	"github.com/futurehomeno/fimpgo/fimptype"
+)
+
+// Constants defining important properties specific for the service.
+const (
+	MaxLvl = "max_lvl"
+	MinLvl = "min_lvl"
+	SwType = "sw_type" // "on_off" or "up_down"
```

Comment:
Const can be longer and slightly different than its value here. SwitchType says much more than just Sw. My first though was Star Wars, you know.

Also, if we have a definite list of values, like here, put on_off and up_down as consts.